docs/installing-northstar/troubleshooting.md

```diff
@@ -105,24 +101,32 @@ Go to Mods at the bottom of your screen on the main menu, then click Authenticat
 
 ## I can't open the console <a href="#console" id="console"></a>
 
-* Navigate to your Titanfall2 directory then open
+* Navigate to your [titanfall2 directory](troubleshooting.md#game-location)
 * Open `R2Northstar\mods\Northstar.Client\mod\cfg\autoexec_ns_client.cfg`
   * And change the \` to one of the F keys (for example `F1` / `F2`) (Note: _Only \~ or F1-12 work_) - This _should_ fix it
     * _Restart game!_
 
 ## The Main Menu is blank <a href="#blank-menu" id="blank-menu"></a>
 
-* Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.` / are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods) / included in the release zip).\\
+* Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.` / are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods) / included in the release zip).\
```

Comment:
```suggestion
* Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.` / are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods) / included in the release zip).
```

not sure why there was a trailing backslash (`\`) still there in the first place ^^